StackExRedis/NHibernate.Caches.StackExRedis.Tests/DistributedRedisCacheFixture.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using NHibernate.Cache;
+using NHibernate.Caches.Common.Tests;
+using NHibernate.Caches.StackExRedis.Tests.Providers;
+using NUnit.Framework;
+
+namespace NHibernate.Caches.StackExRedis.Tests
+{
+	[TestFixture]
+	public class DistributedRedisCacheFixture : CacheFixture
```

Comment:
I do not get what is the point of testing `DistributedRedisCache`, which is a test artefact only.

Reply:
This test was intended to demonstrate that the provider can be extended to work in a distributed environment if someone would need it. We can remove the test along with the distributed provider or leave it for reference if someone would ask how to make one. I'm fine with both decisions.

Reply:
Ok, maybe just add a comment in the file stating its purpose, if there is not already one somewhere I had missed.

Reply:
Done.